contents/articles/2017-01_render-visual-field/index.md

```diff
@@ -0,0 +1,22 @@
+--- 
+title: Adapt Rendering to the User's Current Visual Field
+date: Mon Jan 09 2017 17:13:14 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Daniel Pohl et al., propose to use the user's visual field of the eye gaze for render optimizations and increase performance..." 
+featured_img: "../../../../media/images/blog/visfield.png"
+featured_img_thumb: "../../../../media/images/blog/thumb/visfield.png" 
+---
+
+<img src="../../../../media/images/blog/visfield.png" class='Feature-image u-padBottom--1' alt="Adapt Rendering to the User's Current Visual Field">
+Image Source: [Concept for Using Eye Tracking in a Head-Mounted Display to Adapt Rendering to the User's Current Visual Field](https://perceptual.mpi-inf.mpg.de/wp-content/blogs.dir/12/files/2016/11/pohl2016_vrst.pdf)
+
+<br>
+Advances in head-mounted displays for further immersion increases the need for performance optimizations to handle rendering workload. [Daniel Pohl](http://www.intel.com/content/www/us/en/research/people/intel-labs-bio-daniel-pohl.html) et al., propose to use the user's visual field of the eye gaze for render optimizations and increase performance.
```

Comment:
Edit:

Advances in head-mounted displays for further immersion increases the need for performance optimizations to handle rendering workload. [Daniel Pohl](http://www.intel.com/content/www/us/en/research/people/intel-labs-bio-daniel-pohl.html) et al., propose to use the user's visual field of the eye gaze to optimize the area of the virtual scene that is rendered.